fuzz: 1

This tests correctness of the fuzz algorithm. The fuzz 1 must eat away one line
of the suffix context, but it must not eat anything from the prefix context.
One can imagine that it instead eats one virtual "out-of-file" prefix line.
--- file5.in	2019-01-16 15:02:37.016021405 +0100
+++ with_fuzz1_at_start_mismatch.out	2019-01-16 15:03:08.724512747 +0100
@@ -1,6 +1,6 @@
 bbb mismatched line
 ccc
-ddd
+ddd modified
 eee
 fff
 ggg
